image.pollinations.ai/cloudflare-cache/deploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
@ElliotEtag there seem to be 3 scripts that do similar things
<img width="441" alt="Screenshot 2025-03-21 at 10 33 24" src="https://github.com/user-attachments/assets/9e55934c-c5dd-4e04-8e32-865e194f3712" />

please check if there is duplicated functionality. i dont think we need a new one